sympy/sets/sets.py

```diff
@@ -364,11 +364,12 @@ def is_subset(self, other):
 
         """
         if isinstance(other, Set):
-            dispatch = getattr(self, '_eval_is_subset', None)
-            if dispatch is not None:
-                ret = dispatch(other)
-                if ret is not None:
-                    return ret
+            for obj, target_obj, method_name in [(self, other, "_eval_is_subset"), (other, self, "_eval_is_superset")]:
```

Comment:
I think that every cases where `_eval_is_superset` is needed can be reduced to the `_eval_is_subset` logically.
At least, if `self` and `other` are same classes, calling this twice may not be needed.

Reply:
>  At least, if self and other are same classes, calling this twice may not be needed.

Correct, but https://github.com/sympy/sympy/pull/17771#issuecomment-549084394